azure-iot-device/setup.py

```diff
@@ -90,6 +90,7 @@
         "futures;python_version == '2.7'",
         "PySocks",
         "win-inet-pton;python_version == '2.7'",
+        "future",
```

Comment:
>future [](start = 9, length = 6)

are we actually importing this?

Reply:
Yeah, not sure what this is

---
In reply to: [461195945](https://github.com/Azure/azure-iot-sdk-python/pull/635#discussion_r461195945) [](ancestors = 461195945)

Reply:
this is to make the built ins import work...if we don't have this it does not work...this was surprising to me as well, but it complains that it does not have builtins

---
In reply to: [461199185](https://github.com/Azure/azure-iot-sdk-python/pull/635#discussion_r461199185) [](ancestors = 461199185,461195945)

Reply:
The future module is 2.3 MB and it's only needed for python 2.7.  You should just convert to an int and catch ValueError. 

---
In reply to: [461941297](https://github.com/Azure/azure-iot-sdk-python/pull/635#discussion_r461941297) [](ancestors = 461941297,461199185,461195945)